Gui/opensim/Scripts/modelBuilder.py

```diff
@@ -0,0 +1,45 @@
+import org.opensim.modeling
+import sys
+
+class modelBuilder():
+	"""Class to contain model building utilities."""
+	@staticmethod
+	# jointType is one of: 
+	# 'PinJoint', 'FreeJoint', 'WeldJoint', 'PlanarJoint', 'SliderJoint', 'UniversalJoint'
+	def connectBodyWithJoint(model, bodyName, jointName, jointType):
+		c = model.getComponent(bodyName)
+		c = modeling.PhysicalFrame.safeDownCast(c)
+		p = model.getGround()
+		module = sys.modules['org.opensim.modeling']
+		JointClass = getattr(module, jointType)
```

Comment:
I think you could just directly use `org.opensim.modeling` as the first argument.